### PR TITLE
remove repo from vat_suck

### PR DESCRIPTION
**Pull request by yaoandrew**

### CODE REVIEW

File: transformers/shared/constants/keys.go [hunk 1]
```diff
@@ -54,6 +54,9 @@ const (
 	DataColumn    event.ColumnName = "data"
 	InkColumn     event.ColumnName = "ink"
 	ArtColumn     event.ColumnName = "art"
+	RadColumn     event.ColumnName = "rad"
 	TabColumn     event.ColumnName = "tab"
+	UColumn       event.ColumnName = "u"
```
Comment (yaoandrew) [inline transformers/shared/constants/keys.go]: @elizabethengelman Are we adding these here now? Was looking at your PR...

File: transformers/test_data/vat_suck.go [hunk 2]
```diff
@@ -53,18 +53,17 @@ var VatSuckHeaderSyncLog = core.HeaderSyncLog{
 	Transformed: false,
 }
 
-var VatSuckModel = shared.InsertionModel{
+var VatSuckModel = event.InsertionModel{
 	SchemaName: "maker",
```
Comment (yaoandrew) [inline transformers/test_data/vat_suck.go]: @elizabethengelman I'll rebase when your PR with the new constants gets merged

File: transformers/events/vat_suck/converter.go [hunk 3]
```diff
@@ -20,18 +20,20 @@ import (
...
 	"github.com/makerdao/vulcanizedb/pkg/core"
+	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres"
 )
 
-type VatSuckConverter struct{}
+type Converter struct{}
 
 const (
 	logDataRequired   = false
 	numTopicsRequired = 4
```
Comment (elizabethengelman) [inline transformers/events/vat_suck/converter.go]: There are some shared constants for these vars in https://github.com/makerdao/vdb-mcd-transformers/blob/staging/transformers/shared/converter.go#L28.

File: transformers/shared/constants/keys.go (hunk 1 above)
Comment (elizabethengelman) [reply transformers/shared/constants/keys.go]: Yep!

**Pull request merged**